src/test/components/ActiveTabTimeline.test.js

```diff
@@ -93,7 +93,7 @@ describe('ActiveTabTimeline', function() {
       if (track.type !== 'tab') {
         throw new Error('Expected a tab track.');
       }
-      const threadIndex = track.mainThreadIndex;
+      const threadIndex = (track.threadIndexes.values().next().value: any);
 
       if (threadIndex !== null) {
```

Comment:
Suggestion: I believe this check is now wrong. `threadIndex` may be `undefined`.